map/src/util/index.ts

```diff
@@ -15,3 +15,11 @@ export const isReferrerFromBaseSite = (): boolean => {
   }
   return referrerUrl.origin === 'https://reach4help.org';
 };
+
+export const isInFrame = (): boolean => {
+  try {
+    return window.self !== window.top;
+  } catch (_) {
+    return false;
```

Comment:
```suggestion
    return true;
```

This should be `true` not `false`, as it will fail when `top` and `self` do not have the same origin, and hence are not equal.